### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by aaronpear**
## Description

This PR serves to update the github readme to more accurately reflect the current state of the backend codebase. The previous version of the readme contained numerous inaccurate and or outdated endpoints as well as inaccuracies regarding the overall data structures of the database.

## Loom Video

https://www.loom.com/share/770821cf02c34a5995ca5c4c86929216

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,249 +1,263 @@
...
 <h1>Profiles</h1>
 
 ```
 {
-  key: INCREMENT,
-  okta: STRING,
-  name: STRING,
-  email: STRING,
-  avatarUrl: STRING,
+  profile_id: INCREMENT (primary key, auto-increments, generated by database),
```
Comment (jmerz826) [inline README.md:7]: good consistent formatting for different tables in the schema

File: README.md [hunk 2]
```diff
@@ -1,249 +1,263 @@
...
+  okta_id: STRING (unique),
+  name: STRING (required),
+  email: STRING (required),
+  role_id: INTEGER (required, foreign key),
+  avatarUrl: STRING (required, defaults to: 'https://i.stack.imgur.com/frlIf.png'),
 }
 ```
 
-<h1>Admins</h1>
+| Method   | URL                         | Description                                                                                                              |
```
Comment (jmerz826) [inline README.md:16]: good display of CRUD methods

**Review by jmerz826 — APPROVED** [review]
nice job, the readme needed an update and you did just that.

File: README.md [hunk 3]
```diff
@@ -1,249 +1,263 @@
...
+#### User:
+<p>These endpoints are user-focused. As opposed to the more flexible Profile endpoints where <b>profile_id</b> must be specified, these endpoints retrieve data specific only to the user profile that is making the API request by using the logged-in user's <b>profile_id</b>.</p>
 
-```
-{
-  id: INCREMENT,
-  user_id: INTEGER,
-}
-```
+| Method | URL              | Description                                                             |
```
Comment (moua0061) [inline README.md:28]: I like how you set this up for users to read. As a visual learner, this helps anyone understand the CRUD easier and what it returns.

File: README.md [hunk 4]
```diff
@@ -1,249 +1,263 @@
...
+| [GET]  | /user/schedules/ | Returns an event object with the specified `okta`.                      |
+| [PUT]  | /user/           | Returns an event object with the specified `id`. Updates specific user. |
 
 <h1>Parents</h1>
 
 ```
 {
-  id: INCREMENT,
-  user_id: INTEGER,
+  parent_id: INCREMENT (primary key, auto-increments, generated by database),
```
Comment (moua0061) [inline README.md:39]: Good job on adding the constraints and renaming the id to be more specific.

File: README.md [hunk 5]
```diff
@@ -1,249 +1,263 @@
...
 <h1>Children</h1>
 
 ```
 {
-  id: INCREMENT,
-  user_id: INTEGER,
-  parent_id: INTEGER,
-  username: STRING,
-  age: INTEGER,
+  child_id: INCREMENT (primary key, auto-increments, generated by database),
```
Comment (moua0061) [inline README.md:53]: Same here. Awesome job on renaming the id to be more specific with the user type.

**Review by moua0061 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,249 +1,263 @@
...
+| [GET]    | /profile/:okta_id/       | Returns the profile object with the specified `okta_id`.                                                                       |
+| [GET]    | /profiles/users/:profile_id <b>(BUG: /profiles/users route does not exist; app.js only connects to /profiles/user)</b> | Returns an array filled with event objects that contains information based on profile_id and role_id.                    |
+| [GET]    | /profile/role/:role_id <b>(BUG: does not return any data)</b>      | Returns an array filled with event objects that contain information based on role_id for all profiles of a role_id type. |
+| [POST]   | /profile/                   | Requires a name, password, and email. Registers a new user.                                                              |
+| [PUT]    | /profile/                   | Returns an event object with the specified `okta`. Updates specific profile.                                             |
+| [DELETE] | /profile/:okta_id/             | Returns an event object with the specified `okta`. Deletes specific profile.                                             |
+#### User:
```
Comment (sagun1151) [inline README.md:25]: Great format of different methods and love it telling me which ones has bugs

File: README.md [hunk 7]
```diff
@@ -1,249 +1,263 @@
...
+| Method   | URL                       | Description                                                                        |
+| -------- | ------------------------- | ---------------------------------------------------------------------------------- |
+| [GET]    | /children                  | Returns an array containing all existing children.   
+| [GET]    | /children/:id | Returns the child with the given 'id'.   
+| [GET]    | /children/:id/enrollments | Returns an array filled with event objects with the specified `id`.                |
+| [POST]   | /children/:id/enrollments | Returns the event object with the specified `id`. Enrolls a student.               |
+| [PUT]    | /children/enrollments/    | Returns the event object with the specified `id`. Updates a student's enrollments. <b>(Not Implemented)</b>|
+| [DELETE] | /children/enrollments/:id | Returns the event object with the specified `id`. Unenrolls student from course. <b>(Not Implemented)</b>|
+
+
```
Comment (sagun1151) [inline README.md:70]: Again great formatting

File: README.md [hunk 8]
```diff
@@ -1,249 +1,263 @@
...
+  instructor_id: INTEGER (required, foreign key),
+  program_id: INTEGER (required, foreign key),
+  start_time: TIME (required),
+  end_time: TIME (required),
+  start_date: DATE (required),
+  end_date: DATE (required),
+  location: STRING (required),
+  number_of_sessions: INTEGER (required),
 }
 ```
```
Comment (sagun1151) [inline README.md:127]: Like the addition of saying which keys are required to have a value

**Pull request merged**